.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
+- [ ] Code reviewed
+- [ ] Code tested
+- [ ] User documentation updated
+- [ ] Test plan template updated
+- [ ] Clean commits
```

Comment:
This one is probably more obvious, but is the PR maker going to check this, or the reviewer?